Appraisals

```diff
@@ -446,6 +446,7 @@ elsif Gem::Version.new('2.3.0') <= Gem::Version.new(RUBY_VERSION) \
       gem 'dalli'
       gem 'delayed_job'
       gem 'delayed_job_active_record'
+      gem 'ruby-prof', '< 1.0'
```

Comment:
I had to add this for the `contrib` appraisal to work. `ruby-prof` 1.0 [requires](https://rubygems.org/gems/ruby-prof/versions/1.0.0) ruby `>=2.4.0`.